backend/src/test/scala/cromwell/backend/validation/RuntimeAttributesValidationSpec.scala

```diff
@@ -184,7 +184,7 @@ class RuntimeAttributesValidationSpec
         case Valid(_) => fail("A failure was expected.")
         case Invalid(e) =>
           assert(
-            e.head == "Expecting continueOnReturnCode runtime attribute to be either a Boolean, a String 'true' or 'false', or an Array[Int]"
+            e.head == "Expecting returnCodes runtime attribute to be either a Boolean, a String 'true' or 'false', or an Array[Int]"
```

Comment:
Should this be reverted or reworded?

Reply:
It should be reworded, the next PR will handle that!